src/main/java/competition/subsystems/shooterwheel/ShooterWheelSubsystem.java

```diff
@@ -26,7 +26,8 @@ public ShooterWheelSubsystem(CommonLibFactory factory, PropertyFactory pf, Ideal
         spinWheelPowerProp = pf.createPersistentProperty("Spinning Wheel Power", 1);
 
         if(contract.isShooterWheelReady()){
-            this.shooterWheelMaster = factory.createCANTalon(contract.shooterMotorMaster().channel);
+            this.neoMasterMotor = factory.createCANSparkMax(contract.shooterMotorMaster().channel, this.getPrefix(), "ShooterMaster");
+            this.neoFollowerMotor = factory.createCANSparkMax(contract.shooterMotorFollower().channel, this.getPrefix(), "ShooterFollower");
         }
```

Comment:
You also need to have the follower actually follow the master.
`neoFollowerMotor.follow(neoMasterMotor, true)`